src/modules/esl-share/README.md

```diff
@@ -286,13 +286,15 @@ The principle of cascading is similar to CSS variables. The value is searched fr
  - `instance` - static getter that returns shared instance of `ESLShareConfig`
  - `append` - static method that appends single button or group to current configuration
  - `set` - static method that updates the configuration with promise resolved to `ESLShareConfigInit` or promise provider function
+ - `update` - static method that updates items configuration from the list with the specified partial config
  - `buttons` - getter that returns list of all available buttons
  - `groups` - getter that returns list of all available groups
  - `get` - selects the buttons for the given list and returns their configuration
  - `clear` - clears the configuration
  - `getButton` - gets the button configuration for the given name
  - `getGroup` - gets the group of buttons configuration for the given name
  - `append` - updates the configuration with a `ESLShareButtonConfig`(s) or `ESLShareGroupConfig`(s)
+ - `update` - updates items configuration from the list with the specified partial config
```

Comment:
Would it make sense to split list into static and instance sub-lists?

Reply:
It seems to me that it should be consistent for all docs but we have never done it this way.
Proposal to discuss this point.

Reply:
done